tests/integration/framework/process/placement/placement.go

```diff
@@ -164,3 +175,123 @@ func (p *Placement) InitialClusterPorts() []int {
 func (p *Placement) CurrentActorsAPILevel() int {
 	return 20 // Defined in pkg/actors/internal/api_level.go
 }
+
+func (p *Placement) EstablishConn(ctx context.Context) (*grpc.ClientConn, error) {
+	return grpc.DialContext(ctx, "localhost:"+strconv.Itoa(p.port),
+		grpc.WithBlock(),
+		grpc.WithTransportCredentials(grpcinsecure.NewCredentials()),
+	)
+}
+
+func RegisterHost(t *testing.T, ctx context.Context, conn *grpc.ClientConn, msg *placementv1pb.Host, placementMessage chan any, stopCh chan struct{}) {
```

Comment:
Similarly, this should be a member of `Placement` so we need not pass in a `*conn`.